aggregator/.env.example

```diff
@@ -10,7 +10,7 @@ PRIVATE_KEY_AGG=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff
 PRIVATE_KEY_ADMIN=0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d
 TEST_BLS_WALLETS_SECRET=test-bls-wallets-secret
 
-PG_HOST=localhost
+PG_HOST=127.0.0.1
```

Comment:
Docker is a bit weird about `localhost`. I was getting 'connection refused' when using `--net=host` to connect to the postgres container, but for some reason it works with 127.0.0.1. I don't fully understand it, but since `localhost` is meant to be `127.0.0.1`, I'm happy to leave it at 'docker is being difficult here for no good reason'. If someone knows more, I would still be curious though.